GameData/ROCapsules/PartConfigs/Gemini/GeminiFlightPack.cfg

```diff
@@ -0,0 +1,128 @@
+PART
+{
+
+	// --- general parameters ---
+	name = ROC-GeminiFlightPack
+	module = Part
+	author = Frizzank
+
+	// --- asset parameters ---
+	RSSROConfig = True
+	MODEL
+	{
+		model = ROCapsules/Assets/FASA/WingGemini_FP
+		scale = 1.219, 1.219, 1.219
+	}
+	rescaleFactor = 1.0
+	scale = 1.219
+
+	// --- node definitions ---
+	node_stack_top = 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 2
+	node_stack_bottom = 0.0, -0.69, 0.0, 0.0, -1.0, 0.0, 2
+
+	fx_gasBurst_white = 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, decouple
+	sound_vent_large = decouple
+
+	// --- editor parameters ---
+	TechRequired = aerospaceTech
+	entryCost = 38200
+	cost = 5000
+	//Catigorys for parts Pods Propulsion Control Structural Aero Utility Science
+	category = Control
+	subcategory = 0
+	title = Gemini Flight Pack
+	manufacturer = McDonnell Aircraft
+	description = Decoupler, and flight pack for Winged Gemini.	 This pack has a decoupler on the bottom to jettison spent fuel tanks or engines.
+
+	// attachment rules: stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,0,1,1,1
+
+	stackSymmetry = 1
+
+	// --- standard part parameters ---
+	mass = 0.325
+	dragModelType = default
+	maximum_drag = 0.2
+	minimum_drag =0.2
+	angularDrag = 0.205
+	crashTolerance = 15
+	breakingForce = 25
+	breakingTorque = 25
+	maxTemp = 3200
+
+	CoMOffset = 0.0, -0.35, 0.0
+
+
+	stageOffset = 1
+	childStageOffset = 1
+
+	stagingIcon = DECOUPLER_VERT
+
+
+
+	MODULE
+	{
+		name = ModuleSAS
+	}
+
+	MODULE
+	{
+		name = ModuleDecouple
+		explosiveNodeID = bottom
+		isOmniDecoupler = false
+		ejectionForce = 150
+	}
+	//Reflection shader plugin
+	MODULE
+	{
+		name = ReflectiveShaderModule
```

Comment:
Is that needed with TU? Probably doesn't work anyway since that plugin doesn't get included with ROCapsule nor is it a dependency.